photography/index.html

```diff
@@ -1,106 +1,10 @@
-Please make sure you README.md!
--->
-
 <html lang="en-US">
-<head>
-	<meta charset="utf-8">
-	<meta http-equiv="x-ua-compatible" content="ie=edge">
-	<meta name="viewport" content="width=device-width, initial-scale=1">
-	<meta name="author" content="Claire Violet">
-	<meta name="description" content="This space intentionally left blank.">
-	<meta name="keywords" content="Claire Violet, Violet, Seattle, ultraviolet, UV, ultra violet">
-	<title>uv.gay</title>
-	<meta property="og:url" content="https://ultraviolet.gay/">
-	<meta property="og:title" content="claire's photos">
-	<meta property="og:description" content="photography by claire violet">
-	<meta property="og:image" content="preview.png">
-	<meta name="twitter:card" content="summary">
-	<meta name="twitter:site" content="@assert_h">
-	<meta name="twitter:creator" content="@assert_h">
-	<link rel="shortcut icon" href="favicon.ico">
-	<link rel="icon" type="image/png" href="icon.png">
-	<link rel="apple-touch-icon" href="icon.png">
-	<meta name="msapplication-TileColor" content="#440099">
-	<meta name="theme-color" content="#440099">
-	<link rel="stylesheet" href="theme.css">
-	<style>
-		a:hover {
-			text-decoration-color: #fedb00 !important;
-			text-decoration-style: wavy !important;
-			  color: #fedb00 !important;
-		}
-	</style>
-</head>
-<body>
-
-	<r-grid columns=4 columns-s=1>
-		<r-cell span=row>
-			<h1 class="xxxlarge">
-				<span class="highlight">ultraviolet.darkroom</span>
-				<a href="visible.html">☼</a>
-				<!--<a href="invisible.html">☀</a>-->
-			</h1>
-		</r-cell>
-		<r-cell span=2 span-s=row>
-	  		<p>
-				A spectrum spans between ephemeral and archival.
-	  		</p>
-		    <h3>
-				↪ <a href="visible.html">
-				  Photo Wall
-				</a>
-			</h3>
-			<h3>
-				⟶ <a href="https://blog.ultraviolet.gay">
-				  Blog
-				</a>
-			</h3>
-			<h3>
-				🡒 <a href="https://glass.photo/108-c">
-				  Glass
-				</a>
-			</h3>
-			<h3>
-				Special Pages:
-			</h3>
-			<h3>
-				→ <a href="https://claire.jetzt">
-				  About Me
-				</a>
-			</h3>
-			<h3><span class="callout">
-				→ <a href="projects/sensor-noise/5.html">
-				  Sensor Noise, Episode № 5
-				</a></span>
-			</h3>
-			<h3>
-				⟶ <a href="airborne.html">
-				  Flying around the Sound
-				</a>
-			</h3>
-			<h3>
-				🡒 <a href="decommissioned.html">
-				  Satsop Nuclear Power Plant
-				</a>
-			</h3>
-		</r-cell>
-		<r-cell span=3.. span-s=row>
-		  <img src="me.jpg" class="top right" />
-		</r-cell>
-			
-		<r-cell span=3.. span-s=row>
-		  <h4>Post-Script</h4>
-		  <br>
-		  <p>
-			If any of the images show as missing, give the page a gentle refresh or two. Fastmail (the current host) seems to be slower on the initial response than browsers would like, but they'll get to you eventually, I promise. Loading pre-compressed thumbnails is for cowards.
-		  </p>
-		</r-cell>
-	</r-grid>
-</body>
+  <head>
+    <meta
+      http-equiv="Refresh"
+      content="0; url='https://ultraviolet.gay/index.html'"
```

Comment:
This entire domain is now about my photography, I don't need to maintain this page separately anymore.